Source/Core/VideoCommon/FreeLookCamera.h

```diff
@@ -36,50 +57,43 @@ class CameraController
 
   virtual void Reset() = 0;
 
-  virtual void DoState(PointerWrap& p) = 0;
+  void IncreaseFovX(float fov);
+  void IncreaseFovY(float fov);
+  float GetFovStepSize() const;
+
+  void ModifySpeed(float multiplier);
+  void ResetSpeed();
+  float GetSpeed() const;
+
+private:
+  static constexpr float MIN_FOV_MULTIPLIER = 0.025f;
+  static constexpr float DEFAULT_SPEED = 60.0f;
+  static constexpr float DEFAULT_FOV_MULTIPLIER = 1.0f;
+
+  float m_fov_x_multiplier = DEFAULT_FOV_MULTIPLIER;
+  float m_fov_y_multiplier = DEFAULT_FOV_MULTIPLIER;
+  float m_speed = DEFAULT_SPEED;
+  bool m_dirty = false;
 };
 
 class FreeLookCamera
 {
 public:
   FreeLookCamera();
   void SetControlType(FreeLook::ControlType type);
-  Common::Matrix44 GetView();
-  Common::Vec2 GetFieldOfView() const;
-
-  void MoveVertical(float amt);
-  void MoveHorizontal(float amt);
-  void MoveForward(float amt);
-
-  void Rotate(const Common::Vec3& amt);
-  void Rotate(const Common::Quaternion& amt);
-
-  void IncreaseFovX(float fov);
-  void IncreaseFovY(float fov);
-  float GetFovStepSize() const;
-
-  void Reset();
+  Common::Matrix44 GetView() const;
+  Common::Vec2 GetFieldOfViewMultiplier() const;
 
   void DoState(PointerWrap& p);
 
-  bool IsDirty() const;
-  void SetClean();
-
-  void ModifySpeed(float multiplier);
-  void ResetSpeed();
-  float GetSpeed() const;
-
   bool IsActive() const;
 
+  CameraController* GetController() const;
+
 private:
   bool m_dirty = false;
```

Comment:
This field is now unused (a warning is generated on [osx](https://dolphin.ci/#/builders/3/builds/3721) and [bsd](https://dolphin.ci/#/builders/13/builds/3731) builders); it should be removed.

Reply:
Whoops.  Thanks for catching this!